From c5e83f91d2aaf5efca29fde84eb08cc9b8f590a8 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Sun, 5 Oct 2014 14:39:31 -0500
Subject: [PATCH 1306/1543] MLK-9675 ARM: dts: code change for new QSPI chip
 on SDB board

commit 4f4c7996cff51196ac82ab83c1d946a9cbf92eb5 from
git://git.freescale.com/imx/linux-2.6-imx.git

QSPI chip changed from spansion s25fl128s to micron n25q256a

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked from commit 56c87a201946e4582de2f574c218e43c4db7fadb)
(cherry picked from commit 628c08401cf3b6fcd477bdf118269b0c868e16d8)
---
 arch/arm/boot/dts/imx6sx-sdb.dts |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 7c2d018..25d0c81 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -375,19 +375,19 @@
 	pinctrl-0 = <&pinctrl_qspi2_1>;
 	status = "okay";
 
-	flash0: s25fl128s@0 {
+	flash0: n25q256a@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "spansion,s25fl128s";
+		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <0>;
 	};
 
-	flash1: s25fl128s@1 {
+	flash1: n25q256a@1 {
 		#address-cells = <1>;
 		#size-cells = <1>;
-		compatible = "spansion,s25fl128s";
+		compatible = "micron,n25q256a";
 		spi-max-frequency = <29000000>;
 		spi-nor,ddr-quad-read-dummy = <6>;
 		reg = <1>;
-- 
1.7.5.4

